scipy/stats/_qmc.py

```diff
@@ -914,13 +910,38 @@ class LatinHypercube(QMCEngine):
 
         .. versionadded:: 1.8.0
 
+    orthogonal : bool, optional
+        Orthogonal array based LHS of strength 2 [7]_, [8]_.
+        When used, only ``n=p**2`` points can be sampled, with ``p`` a prime
+        number. It also constrains ``d <= p + 1``.
+        Default is False.
+
+        .. versionadded:: 1.8.0
+
     seed : {None, int, `numpy.random.Generator`}, optional
         If `seed` is None the `numpy.random.Generator` singleton is used.
         If `seed` is an int, a new ``Generator`` instance is used,
         seeded with `seed`.
         If `seed` is already a ``Generator`` instance then that instance is
         used.
 
+    Notes
+    -----
+
+    LHS is extremely effective on integrands that are nearly additive [2]_.
+    LHS on :math:`n` points never has more variance than plain MC on
+    :math:`n-1` points [3]_. There is a central limit theorem for LHS [4]_,
+    but not necessarily for optimized LHS.
+
+    LHS is an orthogonal array of strength 1.
+    :math:`A` is called an orthogonal array of strength :math:`t` if in each
+    n-row-by-t-column submatrix of :math:`A`, all :math:`q^t` possible
+    distinct rows occur the same number of times.
```

Comment:
Is `n` the same `n` as above - the number of samples? What is `q`?

Is "array" used in the same sense as "NumPy array" here? E.g. [1, 2, 3, 4] is an array, but it is not an orthogonal array?

Reply:
> Is `n` the same `n` as above - the number of samples? What is `q`? (is it sqrt(n))?

Yes to both. I will re-read the text and make sure all is defined before hand.

> Is the NumPy array returned by the `lhs.random` an example of the type of "array" discussed here?
> ...
> Since `orthogonal=True` produces an "Orthogonal array based LHS of strength 2", I think that `t=2`.
> If `n` is still the number of points (9), then isn't any "n-row-by-t-column submatrix of :math:`A`" the whole array? This array doesn't seem to contain all possible distinct rows.

Here the orthogonal array of strength 2 has been perturbed by a LHS, this is why you do not see directly the distinct rows. To get what you want, you need to remove the perturbation as we do in the tests: `sample =(sample*np.sqrt(3)).astype(int)`. It produces this with your example
```
array([[0, 0],
       [1, 0],
       [2, 0],
       [0, 1],
       [1, 1],
       [2, 1],
       [0, 2],
       [1, 2],
       [2, 2]])
```

Reply:
If possible, briefly describing how an orthogonal array is perturbed to become an "orthogonal Latin hypercube" might help make the distinction more clear. If it is too complicated for a sentence or two, never mind.

Reply:
I tried to clarify this.